### PR TITLE
#77 Consistent image size and UI improve

### PR DESCRIPTION
**Pull request by FakhrulRadziCode**
some fixes and changes I made...

consistent image size (cant find a proper surau image, sorry)
image

pill design for the search bar
image

improve some UI design by including arrow to indicate the go back button 
image

### CODE REVIEW

File: src/pages/search.tsx [hunk 1]
```diff
@@ -124,7 +138,9 @@ const SurauList = () => {
                     {surau.district.name}, {surau.state.name}
                   </p>
                   <div className="mt-2 flex flex-wrap gap-1">
-                    {surau?.is_solat_jumaat && <Badge color="green" text="Solat Jumaat" />}
+                    {surau?.is_solat_jumaat && (
+                      <Badge color="green" text="Solat Jumaat" />
+                    )}
```
[the remark this answers is not in the document]
Comment (farhan-helmy) [reply src/pages/search.tsx:143]: Can ignore other part implementation of && we will fix that one later this should be a good start

**Review by farhan-helmy — CHANGES_REQUESTED** [review]

File: src/pages/search.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (FakhrulRadziCode) [reply src/pages/search.tsx]: done. am i doing this correctly? 😂

[the remark this answers is not in the document]
Comment (farhan-helmy) [reply src/pages/search.tsx]: yessirr tq sir @FakhrulRadziCode terbaikk

**Review by farhan-helmy — APPROVED** [review]

File: src/pages/search.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (FakhrulRadziCode) [reply src/pages/search.tsx]: mantapp

**Pull request merged**